app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -685,7 +695,7 @@ describe("FormAudience", () => {
     for (const [fieldName, expected] of [
       ["firefoxMinVersion", NimbusExperimentFirefoxVersionEnum.NO_VERSION],
       ["firefoxMaxVersion", NimbusExperimentFirefoxVersionEnum.NO_VERSION],
-      ["populationPercent", "0.0"],
+      ["population-percent-input", "0"],
```

Comment:
It's a bit wonky to have all the fields be camel case except this one, shouldn't this be consistent?

Also `input` is a bit vague here since they're all inputs, is there a better word we can use like `text` or something?

Reply:
Good catch, this should be referencing the field itself, `populationPercent`, instead of the test id's. The test id's seem to all be hyphenated (e.g. [tooltip-duration-audience](https://github.com/mozilla/experimenter/blob/9f11aadb742aa008cb9eb59f737ade58fe4c10bf/app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx#L414)) while the fields themselves are 🐫